Sources/PMSuperButton.swift

```diff
@@ -93,7 +93,17 @@ open class PMSuperButton: UIButton {
         
         self.layer.insertSublayer(gradient, below: self.imageView?.layer)
     }
-    
+    //MARK:- Haptic Feedback
+    @IBInspectable open var hapticFeedback:Bool = false {
+        didSet{
+            if #available(iOSApplicationExtension 10.0, *) {
+                if hapticFeedback {
+                    let generator = UINotificationFeedbackGenerator()
+                    generator.notificationOccurred(.success)
+                }
+            }
+        }
+    }
```

Comment:
Dop you tested this implementation on a device?

The implementation looks wrong to me, since you configure the haptic feedback, but it will be fired only when the `hapticFeedback` will become `true` and not when the button will be pressed.

Reply:
I am so sorry, will make the correction soon and push it again, so sorry I didn't realised it until you told me, so sorry!